root/entity/Details.js

```diff
@@ -75,7 +73,7 @@ const Details = ({
         <tr>
           <th>{l('Last updated:')}</th>
           <td>
-            {lastUpdated ? formatUserDate($c.user, lastUpdated) : l('(unknown)')}
+            {entity.last_updated ? formatUserDate($c.user, entity.last_updated) : l('(unknown)')}
```

Comment:
For the record, this works thanks to: https://github.com/metabrainz/musicbrainz-server/blob/cf9270aeea263d200289280a836084c824e63fc4/root/types.js#L321-L323